### ISSUE TITLE
Bsl Shaders Causing Contraptions to Render Weirdly (Create 0.5.1 and I&O Compat v0.2.0)

### ISSUE THREAD

**Issue by yoyo7043** [leon-o/iris-flw-compat#53]
![image](https://github.com/leon-o/iris-flw-compat/assets/36887096/2f738b1d-532b-4058-aea3-b27b72883b72)
![2023-06-26_20 34 43](https://github.com/leon-o/iris-flw-compat/assets/36887096/145229ed-a791-4233-a74a-44e84ee42378)
this happens to my contraption whenever i turn shaders on with this mod on my profile

**Comment by MaximilianHollis** [comment]
@yoyo7043 I noticed that some other users (and myself) with Linux and/or AMD graphics cards are experiencing this issue. Do you happen to have a setup that includes those?

**Comment by yoyo7043** [comment]
yes, but i disabled fly wheel batching with a command and it seemed to fix
